mmedit/models/base_models/base_mattor.py

```diff
@@ -159,7 +159,7 @@ def restore_size(self, pred_alpha: torch.Tensor,
     def postprocess(
         self,
         batch_pred_alpha: torch.Tensor,  # N, 1, H, W, float32
-        data_samples: List[EditDataSample],
+        data_samples: EditDataSample,
     ) -> List[EditDataSample]:
```

Comment:
we can return the prediction as tensor, and the loop can be replaced with tensor-like operation.

Reply:
Predictions `batch_pred_alpha` are already tensor. Since samples in one batch may have the different original shapes, the original code use a loop to handle each sample and resize them to the original shape.